### DISCUSSION TITLE
Search bug when typing acute accent pt-br

### DISCUSSION THREAD

**Discussion by SidneiGiacomozzi** [requarks/wiki#5349] [Error / Bug Report]
We changed the search configuration in our wiki to "Databse - PostgreSQL" the system does not find the search when there is a term with an acute accent.

e.g, in pt-br:

1. "**é** só utilizar o comando do exemplo"

![wiki_search_1 1](https://user-images.githubusercontent.com/11852849/171211587-c571c0bc-792d-487f-b3ea-3d8ffbd133e6.png)
![wiki_search_1](https://user-images.githubusercontent.com/11852849/171211599-24828cb9-5940-40e7-a74c-15acda2a1d53.png)

2. "Para resolver o problema **é**"

![wiki_search_2 1](https://user-images.githubusercontent.com/11852849/171211576-de68877a-5a2e-4202-bc9e-a50d5911394f.png)
![wiki_search_2](https://user-images.githubusercontent.com/11852849/171211596-6859f6b7-f26f-496f-a511-48d8e1c3c653.png)

3. "Para resolver o problema **é** só utilizar"

![wiki_search_3 1](https://user-images.githubusercontent.com/11852849/171211594-e437a638-60af-41cf-bd83-efb25bcd675d.png)
![wiki_search_3](https://user-images.githubusercontent.com/11852849/171211591-7d026e34-949f-43c6-b721-a5c771294102.png)

**Comment by NathanAlcantara** [comment]
Even using "Elasticsearch" we find the same situation.
